extconf.rb

```diff
@@ -68,8 +68,8 @@ def rule(target, clean = nil)
 \tdone;
 "
    if clean != nil
-      wr << "\t@-rm tmp/* tests/tmp/* 2> /dev/null\n"
-      wr << "\t@rm Makefile\n" if clean
+      wr << "\t-rm -f tmp/* tests/tmp/* 2> /dev/null\n"
```

Comment:
What is up with the minus before `rm`? I know it was there before as part of `@-rm` but should it really be there?

Reply:
Minus means "ignore non-zero exit status". But with `-f` it shouldn't be needed, let's drop that.